rdrf/rdrf/helpers/notifications.py

```diff
@@ -0,0 +1,49 @@
+from dataclasses import dataclass
+from typing import List
+
+from django.conf import settings
+from django.template.loader import get_template
+from django.utils.translation import ugettext as _
+
+
```

Comment:
I am not sure about all these dataclasses. 
Do we really need them?
They just duplicate the definition we already have in the models.

Can't we just have code that creates the right notifications by creating models?

Reply:
I don't think we need the customisability provided by specifying the NOTIFICATIONS in settings.

The EmailTemplates we could create using datasets in "init_data", similar as we do for users/groups etc.

Reply:
Ok, we can remove the create notifications action and add notifications in the `initial_data` module